docs/src/main/paradox/docs/delta/api/views/composite-sink.md

```diff
@@ -0,0 +1,163 @@
+# Composite Sinks
+
+## Single Composite Sink
+
+By default Nexus Delta will use the Single Composite Sink.
+
+## Batch Composite Sink
+
+@@@ note { .warning }
+
+We recommend to start using Composite Views with the default @ref:[Single Composite Sink](#single-composite-sink). Once
+you have a good understanding of it, the Batch Composite Sink can be used to enhance the performance of your deployment.
+
+@@@
+
+Starting with Delta 1.9, it is possible to configure Nexus Delta to use a Batch Composite Sink. This implementation of
+the Composite Sink can query the Blazegraph common namespace for multiple resource IDs at the same time.
+
+### Configuring the Batch Composite Sink
+
+In order to enable the Batch Composite Sink, configure the following Nexus Delta property
+
+`plugins.composite-views.sink-config = batch`
+
+Furthermore, you can configure the maximum size of a batch and the maximum interval using
+
+`plugins.{plugin}.batch.max-elements={max-elements}`
```

Comment:
Those are the default values but it should be possible to tune them at the composite views level without affecting the ES and BG level no ?

Reply:
Updated